app/views/campaigns/toggle_archived.turbo_stream.erb

```diff
@@ -0,0 +1 @@
+<%= turbo_stream.update @campaign %>
```

Comment:
the `turbo_stream.xxx` helper will use `dom_id` when you pass it a model, so this short hand for:

`turbo_stream.update dom_id(@campaign), partial: "campaigns/campaign", locals: { campaign: @campaign }